homeassistant/components/august/lock.py

```diff
@@ -13,7 +14,7 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-SCAN_INTERVAL = timedelta(seconds=10)
+SCAN_INTERVAL = timedelta(seconds=5)
```

Comment:
Does this mean that we hit their API this often? That is too often.

Reply:
No change in the throttle, there could be newer data from the activity update that the device won't see unless it does a check.  I'll actually be able to get rid of all the frequent doorbell polls soon after this merges: https://github.com/snjoetw/py-august/pull/40

Reply:
Ok.

Would it be possible for pyaugust to notify Home Assistant when new data is available ?

Reply:
Ideally the activity poll happens periodically and we can lower the raise the scan interval to 1800 to get the battery state and have activity updates tell the entity to update when new activity comes in.  Figuring out how to get a periodic update of activity that isn’t tied to an entity and just the August component itself is what what I am stuck on

Reply:
This is what the DataUpdateCoordinator does. You have it poll the activity feed and send updates to registered entities.

Reply:
Is there a way to tell it to only update a specific listener ?  It looks like its going to update all of the entities when there is activity for a single one.

Alternatively I could have a call that searches the entity registry to map the deviceid which is the unique to the entity and then update it

Reply:
No, that's not possible. You would have to create your own version of the update coordinator that can distribute to the right devices. For example something like this https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/ring/__init__.py#L238

Reply:
Thanks for that.  I think I can work out a paired down version that will do what is needed.